### PR TITLE
Added ABILITY_TAME to the If include keeping sneak/invis on

### PR DESCRIPTION
**Pull request by Wiggo32**
Takes care of https://github.com/DarkstarProject/darkstar/issues/4281

### CODE REVIEW

File: src/map/entities/charentity.cpp [hunk 1]
```diff
@@ -927,7 +927,7 @@ void CCharEntity::OnAbility(CAbilityState& state, action_t& action)
         }
 
         // remove invisible if aggresive
-        if (PAbility->getID() != ABILITY_FIGHT)
+        if (PAbility->getID() != ABILITY_TAME || PAbility->getID() != ABILITY_FIGHT)
```
Comment (bendangelo) [inline src/map/entities/charentity.cpp]: This should be `&&` right? If the ability is not either then remove invisible.

Comment (Wiggo32) [reply src/map/entities/charentity.cpp]: This is the exact coding that kj told me to write and PR.  Ask him if && would be better...

**Comment by teschnei** [comment]
Yes, it should be and

**Comment by Wiggo32** [comment]
I dont know what I did... im making a new one...

**Pull request closed without merging**